core/src/test/java/org/bptlab/cepta/data_to_database/DataToDatabaseTests.java

```diff
@@ -0,0 +1,207 @@
+package org.bptlab.cepta;
+
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.concurrent.TimeUnit;
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.apache.flink.streaming.api.datastream.AsyncDataStream;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.datastream.DataStreamUtils;
+import org.bptlab.cepta.models.events.train.LiveTrainDataProtos.LiveTrainData;
+import org.bptlab.cepta.models.events.train.PlannedTrainDataProtos.PlannedTrainData;
+import org.bptlab.cepta.config.PostgresConfig;
+import org.bptlab.cepta.operators.DataToDatabase;
+import org.bptlab.cepta.operators.LivePlannedCorrelationFunction;
+import org.bptlab.cepta.operators.WeatherLocationCorrelationFunction;
+import org.bptlab.cepta.providers.LiveTrainDataProvider;
+import org.bptlab.cepta.providers.PlannedTrainDataProvider;
+import org.bptlab.cepta.providers.WeatherDataProvider;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+import com.google.protobuf.GeneratedMessage;
+
+import sun.awt.image.SunWritableRaster.DataStealer;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.sql.*;
+
+public class DataToDatabaseTests {
+
+    public void initDatabaseStuff(PostgreSQLContainer container) {
```

Comment:
Please change method name with stuff to something better

Reply:
jo